test/CMakeLists.txt

```diff
@@ -216,14 +216,11 @@ if (NGRAPH_UNIT_TEST_OPENVINO_ENABLE)
     message(STATUS "NGRAPH_UNIT_TEST_OPENVINO_ENABLE = TRUE")
     add_definitions(-DNGRAPH_UNIT_TEST_OPENVINO_ENABLE)
 
-    if (ENABLE_MKL_DNN)
-        message(STATUS "NGRAPH_TESTS: CPU enabled")
```

Comment:
I'm aware that this might not be the best solution. However at the moment of writing this I couldn't add to the nGraph another backend with the same name. We already had backends named `CPU` and `GPU`, thus that would conflict. I thought of one solution like `IE:<device_name>` ie: `IE:GPU`, but that appeared to cause some compilation errors (I have explained this in one of the comments above). 

After the recent work of one of our colleagues on enabling this backend in CI it turned out that there is much of work needed to enable building simultaneously multiple backends in nGraph along with `IE` backend (ie: `CPU`, `PlaidML` and `IE`) inside DLDT repository. Thus we would have to come with another solution in the future.

Reply:
The other point that was behind this change was to somehow differentiate this backend from other nGraph backends. That was needed to be able to run unit tests from ONNX library.